bentoml/_internal/io_descriptors/json.py

```diff
@@ -129,13 +137,10 @@ def predict(input_arr: pd.DataFrame):
     def __init__(
         self,
         pydantic_model: t.Type[pydantic.BaseModel] | None = None,
-        validate_json: bool = True,
```

Comment:
Maybe we should leave this in for backward compatibility. We can document the deprecation and raise a warning if the user explicitly set it as `False`.

Reply:
We can remove these minor breakages in the next minor release. Within a minor release, I think we should aim for zero breaking changes.

Reply:
@ssheng updated accordingly, there should be zero breaking change besides a minor type change. Any existing user code should work, and a warning will be printed if user have set `validate_json` in their code.